nautobot/users/views.py

```diff
@@ -293,13 +301,16 @@ def update(self, request, *args, **kwargs):
                 filter_params[key] = request.GET.get(key)
             else:
                 filter_params[key] = request.GET.getlist(key)
+
         if filter_params:
             sv.config["filter_params"] = filter_params
+        elif all_filters_removed:
+            sv.config["filter_params"] = {}
 
         if table_changes_pending:
             table_class = self.get_table_class_string_from_view_name(sv.view)
             if table_class:
-                if sv.config.get("table_config", None):
+                if sv.config.get("table_config", None) is None:
```

Comment:
Yeah it is intended. It is trying to make sure that the cleared views would not throw a `KeyError`, since we are setting the `config` attribute to `{}`